int/config/config.go

```diff
@@ -1,4 +1,4 @@
 package config
 
-//nolint:gochecknoglobals
+// nolint:gochecknoglobals
 var Version = "dev"
```

Comment:
Why we need a global for the version ? Is it the only way to define the version ?